OpenQA/Isotovideo/Utils.pm

```diff
@@ -214,13 +214,21 @@ sub handle_generated_assets ($command_handler, $clean_shutdown) {
             push @toextract, _store_asset($i, $name, $dir);
         }
         if ($bmwqemu::vars{UEFI} && $bmwqemu::vars{PUBLISH_PFLASH_VARS}) {
-            push(@toextract, {pflash_vars => 1,
+            push(
```

Comment:
You broke GH syntax coloring!

![image](https://github.com/os-autoinst/os-autoinst/assets/229240/e4cc5489-3271-4be9-8610-3af47dd9bb76)